common/debug_pb_message.cpp

```diff
@@ -43,12 +34,7 @@ void LimitedPrinter::PrintString(const std::string &val,
                                  ::google::protobuf::TextFormat::BaseTextGenerator *generator) const
 {
     auto length = val.length();
```

Comment:
this line needs to be removed to fix CI

Reply:
Yeah I'm still playing around with this PR, seeing what's up. I've done a bit of research on the front and I want to see if I can maintain compatibility with what we've done so far